README.md

```diff
@@ -275,11 +275,13 @@ print(3 / 0)! // undefined
 
 ## Strings
 
-Strings can be declared with single quotes or double quotes.
+Strings can be declared with single quotes, double quotes, backticks, or french quotes.
 
 ```java
 const const name = 'Lu'!
 const const name = "Luke"!
+const const name = `Luke`!
+const const name = «antoine»!
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Guillemet) Guillemet usage varies from language to language. „Guillemets are used pointing outwards («like this») to indicate speech“ in some languages and in others they are „used pointing inwards (»like this«) to indicate speech“.  In french they are „spaced out by [thin spaces](https://en.wikipedia.org/wiki/Thin_space) « like this »“.